### ISSUE TITLE
New 1st Lev Wiz

### ISSUE THREAD

**Issue by Grammarsalad** [UnearthedArcana/B_Spells#137]
### Level 1

**Bleed**  
(Necromancy)

Level: 1
Range: 10’
Duration: Special
Casting time: 1
Saving Throw: Special

Creature suffers 1d6 slash damage, and saves or takes damage each round/lev (max 5 rounds).

**Comprehend Languages**
(Alteration)

Level: 1  
Range: 0  
Casting Time: 1  
Duration: 1 turn  
Area of Effect: The Caster  
Effect: Dialog options (can cast in dialog)

**Enlarge**  
(Alteration)

Explore the possibility (maybe make a 3rd+ level spell)

**Exterminate**  
(Necromancy)

Level: 1
Range: 10’
Duration: Special
Casting time: 1
Saving Throw: None

This spell will instantly kill any animal that it targets, as well as up to 3 additional nearby animals.  Further, the area will clear of any insect swarm like spell, such as creeping doom.  

**Filth’s Bane**  
(Alteration)

Level: 1
Range: Touch
Casting time: 1

Touched creature saves vs. death to be cured of poisons or diseases.

_**Notes**_:
**0-Level Cantrips**: This spell is converted to cantrip if 0-level cantrips is installed.  

**Irritation**  
(Alteration)

Level: 1  
Range: ?  
Casting time: 2  
Area of Effect: 1-4 creatures (as hold person, say)  
Saving Throw: None

Affected creatures suffer a -2 penalty to AC, and suffer spell failure of 10%. P469
An irritation spell affects the epidermis of the subject creatures. Creatures with very
thick or insensitive skins (such as buffalo, elephants, scaled creatures, etc.) are basically
unaffected. There are two versions of the spell, either of which can be cast from the
standard preparation:
Itching. When cast, this causes each subject to feel an instant itching sensation on some
portion of its body. If one round is not immediately spent scratching the irritated area, the
creature is so affected that the next three rounds are spent squirming and twisting,
effectively worsening its Armor Class by 4 and its attack rolls by 2 during this time. Spell
preparations are ruined in the first round this spell is in effect, but not in the following
three rounds. Doing nothing but scratching the itch for a full round prevents the rest of
the effect. If cast at one creature, the saving throw has a -3 penalty; if cast at two
creatures, the saving throw has a -1 penalty; and if cast at three or four creatures, the
saving throw is normal.
Rash. When a rash is cast, the subject notices nothing for 1d4 rounds, but thereafter its
entire skin breaks out in red welts that itch. The rash persists until either a cure disease or
dispel magic spell is cast upon it. It lowers Charisma by 1 point per day for each of four
days (i.e., maximum Charisma loss is 4 points). After one week, Dexterity is lowered by
1 point also. Symptoms vanish immediately upon the removal of the rash, and all
statistics return to normal. This can be cast at one creature only, with a saving throw
penalty of -2

**Last Word**  
(Alteration)

Level: 1  
Range: 60 ft. 
Casting time: 1  
Area of Effect: 1 creature  
Saving Throw: Neg  

Affected creatures must save vs. spell or be silenced for 1 round/lev (max 10 rounds)

**Mending**
(Alteration)

Meh: 1st level
Repairs broken items (will only be useful in BG1 unless item repair stuff is in.  See about expanding the idea)

**Protection from Cantrips**
(Abjuration)

Level: 1

**Push**   
(Evocation)  

Level: 1  
Range: 60 ft.  
Duration: Instantaneous  
Casting Time: 1  
Area of Effect: 1 Target  
Saving Throw: Special

Upon casting this spell, the mage causes an invisible force to strike against whatever creature <PRO_HESHE> is pointing at. The target creature is pushed 20 ft. away from the caster and must save vs. breath to avoid falling down for 1 round.  

Huge (and larger) creatures are immune to this spell.  

**Phantasmal Force**  
(Illusion/Phantasm)

Level: 1
Range: Visual Range of Caster
Duration: 2 rounds + 1 round/level
Casting Time: 4
Area of Effect: Special
Saving Throw: Special

This spell creates the illusion of fearsome creatures that 'fight' on the caster's behalf.  

This spell only creates a basic fascimilie of life, and they do not react correctly in combat or when hit, nor are they affected by most spells that affect normal creatures.  As such, enemies can see through them relatively easily.

Any enemy that gets within melee range, or that attacks or hits them, can save vs. spell to disbelieve them.  Any enemy that disbelieves an illusion cannot be affected by that illusion for a round, and can easily destroy them with a hit.

Finally, creatures created by this spell cannot kill.  All damage they do is non-lethal, and they cannot hurt a creature that disbelieves them.

**Protection from Cantrips**  
(Abjuration)

Level: 1
Range: Touch  
Duration:  5 hours 
Casting Time: 9   
Area of Effect: Creature Touched   
Saving Throw: None   

By casting this spell, the wizard receives immunity to the effects of cantrips cast by other wizards, apprentices, or creatures that use the cantrip spell. The spell protects the caster, or one item or person that he touches (such as a spell book or a drawer containing spell components).

**Read Magic**  
(Divination)

Level: 1  
Range: 0  
Duration: 1 turn  
Casting Time: 1  
Area of Effect: The caster   
Saving Throw: None  
 
(Able to use all scrolls)

**Transcription**   
(Divination)  

Level: 1  
Range: 0  
Duration: 8 hours  
Casting Time: 9  
Area of Effect: The caster   
Saving Throw: None  

**Transcription**   
(Divination)  

Level: 1  
Range: 0  
Duration: 8 hours  
Casting Time: 9  
Area of Effect: The caster   
Saving Throw: None  

With this spell, the caster gains sudden insight allowing <PRO_HIMHER> to transcribe a spell that <PRO_HESHE> currently has prepared to a temporary script that can be used by anybody to cast that spell.  When Transcription is cast, the caster selects from among the spells that <PRO_HESHE> currently has memorized, excepting Transcription, sequencers, contingencies, and the identify spell.  That spell is then removed from <PRO_HISHER> memory, and implanted in the script that appears in <PRO_HISHER> inventory.  <PRO_HESHE> can then use the script <PRO_HISHER>self or give it to one of <PRO_HISHER> allies (there are no restrictions on use--all that is required is that the user speak a trigger word that can be easily memorized).  The script functions much like a read scroll, except that the spell is cast at the lowest possible level required to cast it, and the script dissolves if not used within 8 hours.  

Note that Transcription cannot be cast in the presence of enemies.  

_**Notes**_:
**1st Level Cantrips**: This spell is converted to an Second Level Spell unchanged if 1st level cantrips is installed.  

**Snowball**   
(Evocation)  

Level: 1  
Range: 60 ft.  
Duration: Instantaneous  
Casting Time: 1  
Area of Effect: 1 Target  
Saving Throw: 1/2

This spell creates a mystical snowball which flies at any target of <PRO_HISHER> choice within range.  The snowball hits automatically or 1d6 points of cold damage (double against fire using creatures).  This spell does an additional 1d6 points of cold damage every 2 levels of experience (maximum 5d6 cold damage).

**Comment by Grammarsalad** [comment]
Illusion: Like a summon, but with more powerful cre but that does (mostly) non-lethal dam and low HP
More 'real' illusions at higher levels

1) Illusions at a given level are 'stronger' than would be expected for that level. I think you said this somewhere Subtledoctor, but they should be about equal to a summon spell that is two levels higher. 

2) Summoned creatures are 'dispelled' by certain divination magics. I think that the specifics should vary based on the spell, but True Seeing should dispel all creatures summoned by 6th level and below illusionists, and get to save each round or be dispelled (or, maybe reverse it, and allow the caster to dispel them if they save). 

3) I think that 'disbelief' should be handled based on the spell level. Illusions become more complex at higher levels, and presumably this makes them more difficult to disbelieve. Further, many shadow versions actually gain a semblance of reality from 'shadowstuff' or whatever. As such, it should be relatively easy to disbelieve a 1st level phantasmal force, and increasingly difficult to disbelieve a higher level pf. 

My thinking on low level PF is to give them 1hp. Basically, you swing your sword at them, it passes through without reaction from the creature, and so you and your friends automatically disbelieve it. Additionally, they can be disbelieved when attacked as below (perhaps even with a bonus to the 324 save). 

Higher level illusory creatures can be disbelieved when attacked. I was thinking of something like this:

Spell A: destroys the illusory creatures created from a particular illusion spell. 
Spell B: protects the illusory creature from Spell A. These illusory creatures all have SpellB applied to them on creation, and perhaps at each new round. 
Spell C: Removes Spell B from all illusions from a particular spell in the area
Spell D: Effect 1: 324 protecting against Spell D, but with a save vs. spell to avoid. Effect 2: 146 that casts spell C. 
Spell E: Casts spell A after a 1 second delay at self. 

Illusory creatures have 2 Cast Spell on Condition spells that are triggered when they are attacked. 

Condition 1: Target 'nearest enemy' spell 1: Casts Spell D.

Condition 2: Casts Spell E at self

Basically, if a creature is attacked, it will destroy itself if its nearest enemy saves vs. spell (they will see that it is not reacting right to the attack, and will disbelieve the spell). Higher level illusions will be more difficult to disbelieve, and the 324 effect will have a penalty to its save.
